Gopkg.lock

```diff
@@ -519,6 +519,22 @@
   revision = "3af367b6b30c263d47e8895973edcca9a49cf029"
   version = "v0.2.0"
 
+[[projects]]
+  digest = "1:e38ad2825940d58bd8425be40bcd4211099d0c1988c158c35828197413b3cf85"
+  name = "github.com/google/go-github"
```

Comment:
Just an observation for now: this is a fairly big library, moving us from 60MB -> 62MB to the Telegraf binary, on the large side based on what we are doing here.